README.md

```diff
@@ -35,6 +35,20 @@ Collect translatable strings and create `*.pot` files.
 ## check:pot
 Check for common mistakes in translated texts.
 
+## check:spelling
+Checks spelling in `*.md` and `*.html` files. It uses a global custom dictionary
+(file `lib/tasks/spell.dict` in this repository) and supports repository specific
+dictionary (`spell.dict` file in the repository root directory).
+
+Note: The installed aspell English dictionary may differ in different products
```

Comment:
can you do some emphasis on `Note` so it is easily visible? bold or cursive is enough.